disco_aws_automation/disco_deploy.py

```diff
@@ -559,6 +547,66 @@ def handle_blue_green_ami(self, pipeline_dict, ami, old_group,
             self._disco_elb.delete_elb(hostclass, testing=True)
         return False
 
+    def _create_scaling_schedule(self, pipeline_dict, group_name=None, hostclass=None):
```

Comment:
look like this is the meat of it... how about breaking this out to `disco_aws_util.py` or some other comparable file and have it be shared by `disco_deploy.py` and `disco_aws.py`